app/src/hog.c

```diff
@@ -56,6 +56,11 @@ static struct hids_report consumer_input = {
     .type = HIDS_INPUT,
 };
 
+static struct hids_report mouse_input = {
+    .id = 0x04,
```

Comment:
Here is another place where the report constants could be used

Reply:
Indeed. Updated.